web/components/CaretakerTable.tsx

```diff
@@ -0,0 +1,165 @@
+import Link from "next/link";
+
+function CaretakerTable({ columns, data, onDelete }: any) {
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+    useTable({
+      columns,
+      data,
+    });
+
+  const [open, setOpen] = React.useState(Array(data.length).fill(false));
+
+  const metadata = {
+    Address: "address",
+    "Pref. Communication": "prefferedCommunication",
+    "Child Name": "childName",
+    "Household Info": "houseHoldInfo",
+    "Liability Waiver": "liabilityWaiver",
+  };
+
+  return (
+    <div className="flex flex-col">
+      <div className="sm:-mx-6 lg:-mx-8">
+        <div className="inline-block sm:px-6 lg:px-8 w-full">
+          <div>
+            <table {...getTableProps()} className="w-full">
+              <thead>
+                {headerGroups.map((headerGroup) => (
+                  <tr {...headerGroup.getHeaderGroupProps()}>
+                    <th></th>
+                    {headerGroup.headers.map((column) => (
+                      <th
+                        scope="col"
+                        className="py-3 px-6 text-base font-normal tracking-wider text-slate-500 text-center"
+                        {...column.getHeaderProps()}
+                      >
+                        {column.render("Header")}
+                      </th>
+                    ))}
+                  </tr>
+                ))}
+              </thead>
+              <tbody {...getTableBodyProps()}>
+                {rows.map((row, i) => {
+                  prepareRow(row);
+                  return (
+                    <>
+                      <tr
+                        onClick={() => {
+                          setOpen((prevOpen) => {
+                            const newOpen = [...prevOpen];
+                            newOpen[i] = !newOpen[i];
+                            return newOpen;
+                          });
+                        }}
+                        className="cursor-pointer"
+                        {...row.getRowProps()}
+                      >
+                        <td className="border-t">
+                          <RiArrowDropDownLine
+                            className={`text-2xl duration-300 cursor-pointer ${
+                              open[i] && "rotate-180"
+                            }`}
+                          />
+                        </td>
+                        {row.cells.map((cell) => {
+                          return (
+                            <>
+                              <td
+                                className="py-4 px-6 text-base border-t font-normal text-black whitespace-nowrap"
+                                {...cell.getCellProps()}
+                              >
+                                {cell.column.id === "assigned" ? (
+                                  cell.value ? (
+                                    <div className="text-xs bg-blue-200 text-center rounded-md p-2">
```

Comment:
Use `#FFE3B3` and `#DEEBFF` from the Figma instead of `orange-200` and `blue-200`, if a color doesn't already exist in the codebase you can add it to `tailwind.config.js` so it can be reused!